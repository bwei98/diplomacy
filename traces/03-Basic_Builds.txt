# Spring 1901
F Lon - NTH; A Lvp - Yor; F Edi - NWG
A Mar - Gas; A Par - Pic; F Bre - MAO
A Mun - Bur; F Kie - Hol; A Ber H
# Fall 1901
A Bur - Par