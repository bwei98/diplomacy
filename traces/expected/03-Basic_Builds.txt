# Spring 1901
F NTH; A Yor; F NWG
Lvp; Edi; Lon
F MAO; A Pic; A Gas
Par; Bre; Mar
F HOL; A Ber; A Bur
Kie; Ber; Mun
F Stp; A Mos; A War; F Sev
Mos; Stp; Sev; War
F Tri; A Vie; A Bud
Vie; Tri; Bud
A Ven; A Rom; F Nap
Rom; Ven; Nap
A Smy; F Ank; A Con
Smy; Ank; Con
#
# Fall 1901
F NTH; A Yor; F NWG
Lvp; Edi; Lon
F MAO; A Pic; A Gas
Bre; Mar
F Hol; A Ber; A Par
Kie; Ber; Mun; Hol; Par
F Stp; A Mos; A War; F Sev
Mos; Stp; Sev; War
F Tri; A Vie; A Bud
Vie; Tri; Bud
A Ven; A Rom; F Nap
Rom; Ven; Nap
A Smy; F Ank; A Con
Smy; Ank; Con
#
# Spring 1902
F NTH; A Yor; F NWG
Lvp; Edi; Lon
A Pic; A Gas
Bre; Mar
F Hol; A Ber; A Par; F Kie; A Mun
Kie; Ber; Mun; Hol; Par
F Stp; A Mos; A War; F Sev
Mos; Stp; Sev; War
F Tri; A Vie; A Bud
Vie; Tri; Bud
A Ven; A Rom; F Nap
Rom; Ven; Nap
A Smy; F Ank; A Con
Smy; Ank; Con
#
# END
